conductr_cli/sandbox_stop.py

```diff
@@ -1,13 +1,7 @@
-from conductr_cli import sandbox_common, terminal
-from conductr_cli.screen_utils import headline
-import logging
+from conductr_cli import sandbox_stop_docker
 
 
 def stop(args):
     """`sandbox stop` command"""
 
-    log = logging.getLogger(__name__)
-    running_containers = sandbox_common.resolve_running_docker_containers()
-    if running_containers:
-        log.info(headline('Stopping ConductR'))
-        terminal.docker_rm(running_containers)
+    sandbox_stop_docker.stop(args)
```

Comment:
The `sandbox_stop_docker.stop(args)` function is also classed from the ConductR JVM version: https://github.com/typesafehub/conductr-cli/blob/1966fb215f4d00823d0c67f519cd0e9987ce1a3f/conductr_cli/sandbox_run_jvm.py#L22